crates/humanode-peer/src/service/import_queue.rs

```diff
@@ -0,0 +1,81 @@
+//! Build the import queue depending on either (sealing) or (babe, grandpa) cases.
+
+use std::sync::Arc;
+
+use fc_consensus::FrontierBlockImport;
+use sc_consensus_babe::BabeLink;
+use sc_service::{Error as ServiceError, TaskManager};
+use sc_telemetry::TelemetryHandle;
+
+use super::{
+    inherents, Block, FrontierBackend, FullBabe, FullBoxBlockImport, FullClient, FullGrandpa,
+    FullSelectChain,
+};
+
+/// Build the import queue for sealing case.
+pub fn sealing(
+    client: Arc<FullClient>,
+    config: &sc_service::Configuration,
+    task_manager: &TaskManager,
+    frontier_backend: Arc<FrontierBackend>,
+) -> (
+    sc_consensus::DefaultImportQueue<Block, FullClient>,
+    FullBoxBlockImport,
+) {
+    let block_import = FrontierBlockImport::new(
+        Arc::clone(&client),
```

Comment:
The first argument here is usually `babe_block_import` - why don't we also pass it here? It is available at the callsite at the `service:: new_partial`, so passing is easy - so I assume there must be a reason. Also, if there is a reason - why do we still needlessly create the babe block import at the callsite of this `sealing` fn at service?